File: 程式語言 V2答案檔/C/106.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main ()
{
	int x1, y1, x2, y2; 
    scanf("%d", &x1);
    scanf("%d", &y1);
    scanf("%d", &x2);
    scanf("%d", &y2);
    
    double x = pow(x1-x2, 2);
	double y = pow(y1-y2, 2);
	
	double result = sqrt(x+y);
	int result_2 = (result+0.005)*100;
	
	printf("%.2f", result_2/100.);
 	return 0;
}

File: 程式語言 V2答案檔/PHP/106.php
<?php
    $x1 = intval(fgets(STDIN));
    $y1 = intval(fgets(STDIN));
    $x2 = intval(fgets(STDIN));
    $y2 = intval(fgets(STDIN));
    printf("%.2f\n", sqrt(pow(($x2-$x1), 2) + pow(($y2-$y1), 2)));
?>